contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -135,9 +135,7 @@ static bool check_server_collation_name(char **newval, void **extra, GucSource s
 			* We are storing value in lower case since
 			* Collation names are stored in lowercase into pg catalog (pg_collation).
 			*/
-		char *dupval = pstrdup(*newval);
-		strcpy(*newval, downcase_identifier(dupval, strlen(dupval), false, false));
-		pfree(dupval);
+		strncpy(*newval, downcase_identifier(*newval, strlen(*newval), false, false),strlen(*newval));
```

Comment:
Instead of calling strlen twice, call it once and pass stored len value

Reply:
okay